web/modules/custom/dpl_url_proxy/src/DplUrlProxyInterface.php

```diff
@@ -0,0 +1,15 @@
+<?php
+
+namespace Drupal\dpl_url_proxy;
+
+/**
+ * Interface for dpl_url_proxy constants.
+ */
+interface DplUrlProxyInterface {
```

Comment:
Consider if we can come up with a better name suffix than `Interface` for interfaces.

It does not provide any value and we do not intend it to be used as a "normal" interface which other classes can implement anyway.

Reply:
I get your point about the "mis"use of an interface here.
The thing is I think I started solving the issue about having constants stored somewhere in code like this after I tried doing it the old D7 way with just including them in a module file. But then phpcs was complaining and saying this was the way to do it. 
Another solution could be to create a `final class`.